client/packages/android/app/src/main/java/org/openmsupply/client/MainActivity.java

```diff
@@ -16,7 +17,9 @@ protected void onCreate(Bundle savedInstanceState) {
 
         String path = getFilesDir().getAbsolutePath();
         String cache = getCacheDir().getAbsolutePath();
-        server.start(8000, path, cache);
+        String androidId = Settings.Secure.getString(getContentResolver(),
```

Comment:
Turns out - the header is required, but the value of the id can be blank. I started down the path of adding in a device specific ID, and have left this in place, as it will be useful when the hardware ID aspect of #21 is attended to